client/layout/masterbar/style.scss

```
@@ -1401,3 +1414,53 @@ a.masterbar__quick-language-switcher {
 		opacity: 1;
 	}
 }
+
```

```suggestion
.is-global-sidebar-visible .masterbar__item {
	transition: none;
}

```
Removing transitions to `masterbar__item` looks like this:

https://github.com/user-attachments/assets/5a9e65c2-95b2-427a-8e2b-27103c0c4ecc